parsec/core/resources/default_pattern.ignore

```diff
@@ -0,0 +1,83 @@
+
+
+
+### Temporary extensions ###
+
+*.tmp
+*.temp
+*.swp
+
+
+### Linux ###
+
+# Temporary files
+*~
+
+# Temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
+
+# Glib temporary files
+.goutputstream-*
+
+### macOS ###
+
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
```

Comment:
What does this mean ?

(`Icon` seems like a legit folder name for a regular user...)

Reply:
No idea, I just assumed it was in the list for a reason. I'll remove it.

Reply:
https://stackoverflow.com/a/30755378/2846140

Reply:
e55ed1ec6d0c085d18bfdc8c964be92f342860b4